lua/octo/utils.lua

```diff
@@ -155,26 +159,45 @@ function M.get_remote_name()
     local url = table.concat(job:result(), "\n")
     local stderr = table.concat(job:stderr_result(), "\n")
 
-    if M.is_blank(stderr) then
+    if not M.is_blank(stderr) then
+      goto continue
+    else
+      local host
       local repo
       -- https://github.com/pwntester/octo.nvim.git
       -- ssh://git@github.com/pwntester/octo.nvim.git
       if vim.startswith(url, "http://") or
           vim.startswith(url, "https://") or
           vim.startswith(url, "ssh://") then
         local chunks = vim.split(url, "/")
+        host = chunks[3]
         repo = chunks[4] .. "/" .. chunks[5]
         -- git@github.com:pwntester/octo.nvim.git
       elseif vim.startswith(url, "git@") then
         local parts = vim.split(url, ":")
-        local chunks = vim.split(parts[2], "/")
-        repo = chunks[1] .. "/" .. chunks[2]
+        host = vim.split(parts[1], "@")[2]
+        local repo_chunks = vim.split(parts[2], "/")
+        repo = repo_chunks[1] .. "/" .. repo_chunks[2]
       else
-        return
+        goto continue
       end
-      return string.gsub(repo, ".git$", "")
+      return {
+        host = host,
```

Comment:
```suggestion
        host = vim.split(host, "@")[#vim.split(host, "@")],
```

We need to remove the username from the host chunk

Reply:
Fixed.